analysis/analysis_utils.py

```diff
@@ -227,8 +228,11 @@ def process_results(self,search,image_params,res_filepath,likelihood_level):
         print("Processing Results")
         print('---------------------------------------')
         while likelihood_limit is False:
+            print('Starting pooling...')
             pool = mp.Pool(processes=16)
-            results = search.get_results(res_num,chunk_size)
+            print('Getting results...')
+            if results is None:
+                results = search.get_results(res_num,chunk_size)
```

Comment:
This breaks the loop. You only ever get the results in the first chunk. This needs to be fixed before merging.